auth/src/services/zitadel/parseMetadata.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  Chair,
+  ConferenceAdmin,
+  NonStateActor,
+  Representative,
```

Comment:
Is there a Reason you did choose `representative` instead of `delegate`?

Reply:
This questions has consequences on many files, I won't be commenting on every occasion.

Reply:
Absolutely not, changed this in a recent commit